### PR TITLE
chore: upgrade dependencies to the latest version

### PR DESCRIPTION
**Pull request by ersachin3112**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [x] **metadata update**

### Motivation / Use-Case
upgrade dependencies to the latest version

### Breaking Changes
No

### Additional Info
No

### CODE REVIEW

File: types/linter.d.ts [hunk 1]
```diff
@@ -30,21 +30,21 @@ declare namespace linter {
...
-type Options = import('./options').Options;
-type Compilation = import('webpack').Compilation;
-type Linter = (files: string | string[]) => void;
-type Reporter = () => Promise<Report>;
 type ESLint = import('eslint').ESLint;
 type Formatter = import('eslint').ESLint.Formatter;
 type LintResult = import('eslint').ESLint.LintResult;
 type Compiler = import('webpack').Compiler;
+type Compilation = import('webpack').Compilation;
+type Options = import('./options').Options;
```
Comment (evenstensberg) [inline types/linter.d.ts:38]: why did you move the order here?

**Review by alexander-akait — APPROVED** [review]

File: types/linter.d.ts (hunk 1 above)
Comment (alexander-akait) [reply types/linter.d.ts:38]: I think it done typescript because of update, they are autogenerated
